src/components/graph/menus/modification-list-item.js

```diff
@@ -77,7 +77,9 @@ export const ModificationListItem = ({
         finally, we uses the default value of equipmentId or empty string
     */
     const getComputedLabel = useCallback(() => {
-        const modificationMetadata = JSON.parse(modif.messageValues);
+        const modificationMetadata = modif?.messageValues
+            ? JSON.parse(modif.messageValues)
+            : {};
```

Comment:
If {} is affected to modificationMetadata won't you have some similar trouble in the following switch ? (return modificationMetadata.lineToSplitId etc)